zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/authorization/grpc/DecisionEvaluationEvaluateAuthorizationIT.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.authorization.grpc;
+
+import static io.camunda.zeebe.it.util.AuthorizationsUtil.createClient;
+import static io.camunda.zeebe.it.util.AuthorizationsUtil.createClientGrpc;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.application.Profile;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.ClientStatusException;
+import io.camunda.zeebe.client.protocol.rest.PermissionTypeEnum;
+import io.camunda.zeebe.client.protocol.rest.ResourceTypeEnum;
+import io.camunda.zeebe.it.util.AuthorizationsUtil;
+import io.camunda.zeebe.it.util.AuthorizationsUtil.Permissions;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources.AutoCloseResource;
+import io.camunda.zeebe.test.util.testcontainers.TestSearchContainers;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@AutoCloseResources
+@Testcontainers
+@ZeebeIntegration
+public class DecisionEvaluationEvaluateAuthorizationIT {
```

Comment:
🔧 We should make all these things egine tests. I'm working on a PR to do that for REST atm. Hopefully we can easily reuse that for gRPC. If we have both gRPC and REST in these ITs the job will run into a timeout 😅 

IT-wise we should keep 2 gRPC scenarios. One for basic-auth and one for oidc. I will do the same for REST.

Reply:
Yeah, I'll remove most of the tests. I think we can create gRPC engine tests in a separate PRs.

Reply:
Sounds good to me 👍 If you could please also undo the moving of the rest tests. Because that'll cause conflicts otherwise 😄

Reply:
Yes, I'll remove the commit